website/docs/compose/podman-compose.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Using Podman Compose with Podman Desktop
+---
+
+## Introduction
+
+Podman Desktop is aimed at providing the Podman experience to the developers with a Graphical User Interface. Many developers who enjoy using Podman Engine to work with containers are also avid users of Podman Compose. 
+
+[Podman Compose](https://github.com/containers/podman-compose#readme) is a community-driven tool that allows developers to run multiple containers at the same time and facilitate communication between them. Much like Docker Compose, it requires a Compose YAML file with definitions about the containers that need to communicate.
+
+If you are new to writing [Compose files](https://github.com/compose-spec/compose-spec/blob/master/spec.md#compose-file) or [Compose Spec](https://compose-spec.io/), check [this guide](https://github.com/compose-spec/compose-spec/blob/master/spec.md) for more information. 
+
+## Requirements
+
+### 1. Podman Desktop
+Get the latest version of Podman Desktop by clicking [here](/downloads). Refer to the [installation guide](/docs/installation/macos-install) for more instructions.
+### 2. Podman Compose
+Read along for more information on how to install Podman Compose.
+
+## Get Started
+
+In this tutorial, we will learn about working with containers on Podman Desktop generated using podman-compose. You can find more information about Podman Compose [here](https://github.com/containers/podman-compose/blob/devel/README.md). 
+
+### 1. Install podman-compose
+#### For MacOS
+
+There are many ways to install podman-compose on your MacOS system. One of the easier ways is to use Homebrew. To install podman-compose, simply run the following command
+
+```sh
+brew install podman-compose
+```
+
+To ensure podman-compose is properly installed, run the command 
+```sh
+podman-compose
+``` 
+and it will return more information about the same.
+
+#### For Windows and Linux
+
+Refer this official [installation guide](https://github.com/containers/podman-compose#installation) to get the latest version of Podman Compose on your Windows or Linux system. 
+
+### 2. Using Podman Compose to run multiple containers
+
+For this tutorial, I will use the [flask-redis project](https://github.com/docker/awesome-compose/tree/master/flask-redis) from the [awesome-compose repository](https://github.com/docker/awesome-compose). This repository consists of many sample projects for you to get started with Podman Compose. 
```

Comment:
I feel it is okay to include a line about the repository

Reply:
"This repository consists of many sample projects for you to get started with Podman Compose. " doesn't bring any new information to the user because it's already implied by the link and description of saying "awesome-compose repository"